collect_app/src/main/java/org/odk/collect/android/external/ExternalSQLiteOpenHelper.java

```diff
@@ -263,4 +265,14 @@ private void onProgress(String message) {
             formLoaderTask.publishExternalDataLoadingProgress(message);
         }
     }
+
+    /**
+     * Removes the presence of a Byte Order Mark from the initial UTF-8 String
+     *
+     * @param firstLine String to scan for a Byte Order Marker
+     * @return String with any Byte Order Marker removed
```

Comment:
Thanks for your patience! I have one last question about this comment. You say that you remove any BOM but in fact, you remove only the first one? Is that enough if so please improve this comment removing `any`

Reply:
The BOM only appears at the start of the file and so would only appear as the first Character in the String. I'll improve the comment.